tests/passes/mapping/test_sabre.py

```diff
@@ -38,12 +39,17 @@ def test_simple(compiler: Compiler) -> None:
         GreedyPlacementPass(),
         GeneralizedSabreLayoutPass(),
         GeneralizedSabreRoutingPass(),
+        ApplyPlacement(),
     ]
 
     cc, data = compiler.compile(circuit, workflow, True)
     pi = data['initial_mapping']
     pf = data['final_mapping']
     PI = PermutationMatrix.from_qubit_location(5, pi)
     PF = PermutationMatrix.from_qubit_location(5, pf)
```

Comment:
How did some qudits become inactive? Not sure I understand

Reply:
When you apply the placement, the logical, input circuit size is extended to fit the physical, output machine model. So in this workflow, this means a good portion of them won't be used.